pkg/scanners/trivy/helpers.go

```diff
@@ -12,12 +12,14 @@ func loadConfig(filename string) (Config, error) {
 
 	b, err := os.ReadFile(filename)
 	if err != nil {
+		log.Info("LOADCONFIG unable to read file")
```

Comment:
Maybe `log.Error` here, and remove the `LOADCONFIG` -- the logger should give us more info on where the error is occurring.